src/github.com/getlantern/flashlight/app/settings.go

```diff
@@ -145,6 +145,14 @@ func (s *Settings) checkBool(data map[string]interface{}, name string, f func(bo
 	}
 }
 
+func (s *Settings) checkInt(data map[string]interface{}, name string, f func(int)) {
+	if v, ok := data[name].(int); ok {
```

Comment:
Here's another reference: https://blog.golang.org/json-and-go, please see "Generic JSON with interface{}"